packages/manager/apps/web/client/app/domain/zone/zone.routing.js

```diff
@@ -1,6 +1,30 @@
+import get from 'lodash/get';
 import controller from './domain-zone-dns.controller';
 import template from './domain-zone-dns.html';
 
+const commonResolves = {
+  detachZoneOptions: /* @ngInject */ (
```

Comment:
why is this defined here if it is only used in the detach route. you can still define a common route in detach and use it.

Reply:
@radireddy It is used here https://github.com/ovh/manager/pull/2833/files#diff-09710e5e28a23449aac1f9e2074ee072R32 to check if zone can be detached first